src/modules/git_commit.rs

```diff
@@ -368,7 +368,7 @@ mod tests {
             .output()?;
 
         Command::new("git")
-            .args(&["tag", "v1", "-m", "Testing tags v1"])
```

Comment:
Rather than changing the the existing test can you add an additional test for the new functionality?